lang/java/reef-runtime-local/src/main/java/org/apache/reef/runtime/local/driver/ContainerManager.java

```diff
@@ -164,15 +163,15 @@ public void onNext(final RemoteMessage<ReefServiceProtos.RuntimeErrorProto> valu
       String rackName = it.next().trim();
       Validate.notEmpty(rackName, "Rack names cannot be empty");
       // should start with a separator
-      if (!rackName.startsWith(PATH_SEPARATOR)) {
-        rackName = PATH_SEPARATOR + rackName;
+      if (!rackName.startsWith(Constants.PATH_SEPARATOR)) {
+        rackName = Constants.PATH_SEPARATOR + rackName;
       }
       // remove the ending separator
-      if (rackName.endsWith(PATH_SEPARATOR)) {
+      if (rackName.endsWith(Constants.PATH_SEPARATOR)) {
```

Comment:
You might want to factor out the code to normalize rack names into its own (static) method, e.g. `public static String normalizeRackName(final String rackName)`. There is very similar code to this in `MultiDataCenterEvaluatorToPartitionStrategy`.